wandb/apis/public.py

```diff
@@ -4182,19 +4182,23 @@ def __init__(self, client, entity, project, name, attrs=None):
         self._entity = entity
         self._project = project
         self._artifact_name = name
+        self._artifact_collection_name = name.split(":")[0] if ":" in name else name
```

Comment:
nit: i think `name.split(":")[0]` is satisfactory here

Reply:
added in next commit